src/main/web_src/src/components/ui/dollyTable/pagination/Pagination.js

```diff
@@ -22,16 +24,15 @@ export default class Pagination extends Component {
 		const itemsToRender = this._calculateItemsToRender()
 
 		const startIndex = this._calculateStartIndex() + 1
-		const lastIndex = this._calculateStartIndex() + this.state.itemCount
-		const itemCount = this.props.items.length
-		const renderPagination = itemCount > this.state.itemCount
-		const renderItemCount = ITEM_PER_PAGE < itemCount
+		const lastIndex = this._calculateStartIndex() + this.state.pageSize
+		const itemCount = this.props.gruppeDetaljer?.antallGrupper
```

Comment:
Jeg får bare opp første siden av alle grupper, og ingen paginering 😲  Tror det er fordi den ikke finner `.antallGrupper` og `.items` er bare de 10 første gruppene...? 🤔

Reply:
Whoops, dette er i sammenheng med backend endringer for paginering av identer, har merget inn denne PRen og endret der, men glemte å fikse her 🤪